src/Views/Nav.elm

```diff
@@ -23,7 +23,7 @@ view _ =
                 ]
             ]
         , nav []
-            [ navItem "stories" "#/stories" "story" "view-list" "Stories" "Stories"
+            [ navItem "question-circle" "#/stories" "find-out-more" "view-list" "Find Out More" "Find Out More"
```

Comment:
Does it fit?

Reply:
It is a bit of a squeeze when it gets to an IPhone 5 size it cuts off the exit site text

Reply:
OK Let's make it an acceptance criteria on the issue to address that. I think it'll be easier for @katquatermass to make judgements if we keep pushing stuff to staging.